src/components/FlowRunResults.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <div class="flow-run-results">
+    <div v-if="!noResults" class="flow-run-results__button-group-container">
```

Comment:
Super nit: making the computed `hasResults` would prevent the double negative of `!noResults` and be a bit more readable.